### PR TITLE
refactored debuggerUI

### PR DESCRIPTION
**Pull request by LianaHus**
- using SourceHighlight fix https://github.com/ethereum/remix-ide/issues/2834
- removed self

### CODE REVIEW

File: src/app/tabs/debugger/debuggerUI.js [hunk 1]
```diff
@@ -48,62 +47,62 @@ class DebuggerUI {
...
-        if (path) self.sourceHighlighter.currentSourceLocationFromfileName(lineColumnPos, path)
+        if (path) {
+          await this.debuggerModule.call('editor', 'discardHighlight')
+          await this.debuggerModule.call('editor', 'highlight', lineColumnPos, path)
+        }
       }
     })
 
-    this.debugger.event.register('debuggerUnloaded', self.unLoad.bind(this))
+    this.debugger.event.register('debuggerUnloaded', () => unLoad)
```
Comment (yann300) [inline src/app/tabs/debugger/debuggerUI.js:92]: `unLoad` is this working

Comment (LianaHus) [reply src/app/tabs/debugger/debuggerUI.js:92]: no shouldn't

Comment (LianaHus) [reply src/app/tabs/debugger/debuggerUI.js]: @yann300 pushed another commit

File: src/app/tabs/debugger-tab.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ class DebuggerTab extends ViewPlugin {
   }
 
   deactivate () {
-    this.debuggerUI.unLoad()
+    this.debuggerUI.deletHighlights()
```
Comment (yann300) [inline src/app/tabs/debugger-tab.js:74]: typo `deletHighlights`

Comment (LianaHus) [reply src/app/tabs/debugger-tab.js]: pushed the fix

**Pull request merged**